### DISCUSSION TITLE
Copilot doesn't complete the code

### DISCUSSION THREAD

**Discussion by enygf3** [community/community#13687] [Copilot Conversations]
In the end of the code
(all lines of code on the image below is Copilot's code)
<img width="611" alt="Screenshot 2022-03-28 в 13 24 37" src="https://user-images.githubusercontent.com/78654438/160379033-20dcb7eb-10c1-454b-a104-7149057ad85f.png">
There are not completed line/lines of code. And this problem appears not only with HTML. The same issue I met in pure JS.

**Comment by AhtaN5** [comment]
> In the end of the code
> (all lines of code on the image below is Copilot's code)
> <img alt="Screenshot 2022-03-28 в 13 24 37" width="611" src="https://user-images.githubusercontent.com/78654438/160379033-20dcb7eb-10c1-454b-a104-7149057ad85f.png">
> There are not completed line/lines of code. And this problem appears not only with HTML. The same issue I met in pure JS.
